scripts/config.pl

```diff
@@ -89,6 +93,8 @@
 MBEDTLS_ECP_DP_M221_ENABLED
 MBEDTLS_ECP_DP_M383_ENABLED
 MBEDTLS_ECP_DP_M511_ENABLED
+MBEDTLS_MEMORY_BACKTRACE
```

Comment:
`MBEDTLS_MEMORY_DEBUG` should also be excluded.

Reply:
Addressed in 1263707.